File: fixtures/scale/php/small/src/Style/AnsiPalette.php
<?php

namespace TextPrint\Style;

class AnsiPalette
{
    public const RESET = "\033[0m";
    public const BOLD = "\033[1m";
    public const DIM = "\033[2m";

    public static function colored(string $body, int $code): string
    {
        return "\033[" . $code . 'm' . $body . self::RESET;
    }
}

File: fixtures/scale/csharp/small/Style/AnsiPalette.cs
namespace TextPrint.Style
{
    public static class AnsiPalette
    {
        public const string Reset = "\u001b[0m";
        public const string Bold = "\u001b[1m";
        public const string Dim = "\u001b[2m";

        public static string Colored(string body, int code)
        {
            return "\u001b[" + code + "m" + body + Reset;
        }
    }
}
